src/components/BookContainer.js

```diff
@@ -1,5 +1,6 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/no-typos */
```

Comment:
- [ ] I noticed that you disable a lot **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: